docs/topics/loaders.rst

```diff
@@ -161,10 +92,14 @@ Item Loaders are declared like Items, by using a class definition syntax. Here
 is an example::
 
     from scrapy.loader import ItemLoader
-    from scrapy.loader.processors import TakeFirst, MapCompose, Join
+    from itemloaders.processors import TakeFirst, MapCompose, Join
+
+    from myproject.items import ProductItem
 
     class ProductLoader(ItemLoader):
 
+        default_item_class = ProductItem
+
         default_output_processor = TakeFirst()
```

Comment:
I think this example is not clear after processors docs are removed

Reply:
This one is a bit tricky.
I included a "link" to `itemloaders` doc after the example.
Not sure if it's the best approach